kong/kong.lua

```diff
@@ -252,6 +252,16 @@ function Kong.balancer()
   end
 end
 
+function Kong.rewrite()
+  core.rewrite.before()
+
+  for _, plugin in ipairs(singletons.loaded_plugins) do
+    plugin.handler:rewrite()
```

Comment:
Good point - I will update the PR to only chose plugins where `api_id == nil and consumer_id == nil`.